harness/determined/common/api/request.py

```diff
@@ -106,11 +106,18 @@ def do_request(
     except requests.exceptions.RequestException as e:
         raise errors.BadRequestException(str(e))
 
+    def _get_message(r: requests.models.Response) -> str:
+        try:
+            return str(simplejson.loads(r.text).get("message"))
+        except Exception:
+            pass
```

Comment:
how about `return ""` here instead of pass?

Reply:
Good catch. Done.